src/main/java/uk/ac/cam/cl/dtg/isaac/dao/EventBookingPersistenceManager.java

```diff
@@ -18,6 +18,7 @@
 import uk.ac.cam.cl.dtg.isaac.dto.content.ContentDTO;
 import uk.ac.cam.cl.dtg.isaac.dto.eventbookings.DetailedEventBookingDTO;
 import uk.ac.cam.cl.dtg.isaac.dto.eventbookings.EventBookingDTO;
+import uk.ac.cam.cl.dtg.isaac.dto.users.UserSummaryWithEmailAddressAndGenderDTO;
 import uk.ac.cam.cl.dtg.isaac.dto.users.UserSummaryWithEmailAddressDTO;
```

Comment:
Unused import. How come this wasn't picked up by the pipeline? I wonder if the checkstyle GitHub action only checks the PR changes, rather than the whole project. Maybe we made a mistake with this action, and we should have used a different one, something like Prettier for Java.

Reply:
This is a used import line 300

Reply:
No, I was talking about `UserSummaryWithEmailAddressDTO`, not the new `UserSummaryWithEmailAddressAndGenderDTO`. After your changes, this is no longer used, but I was hoping that the pipeline would have caught the issue.